targets/ESP32/_common/ESP32_C5_DeviceMapping.cpp

```
@@ -39,15 +36,15 @@ int8_t Esp32_SERIAL_DevicePinMap[UART_NUM_MAX][Esp32SerialPin_Max] = {
-    // I2C1 - pins 18, 19,
-    {I2C1_DATA, I2C1_CLOCK}};
+    // I2C1 - no default pins assigned
+    {-1, -1}};
 
 // =============================================
 //  LED PWM
 //  16 channels LED1 to LED16  or PWM1 to PWM16
 //  Map pins Data & Clock
 //  Channels ( non assigned )
-int8_t Esp32_LED_DevicePinMap[TARGET_LED_NUM_PINS] = { -1 };
+int8_t Esp32_LED_DevicePinMap[TARGET_LED_NUM_PINS] = {-1};
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)